src/XMLimport.h

```diff
@@ -52,7 +52,7 @@ class XMLimport : public QXmlStreamReader
 
 public:
     XMLimport(Host*);
-    ~XMLimport() {}
+    virtual ~XMLimport() {}
```

Comment:
My limited understanding is that `virtual` is needed here, because (as https://www.geeksforgeeks.org/virtual-destructor ) puts it:
> Deleting a derived class object using a pointer of base class type that has a non-virtual destructor results in undefined behavior. To correct this situation, the base class should be defined with a virtual destructor.

However I am confused - what is the *derived* class that uses `XMLimport` as the *base* class?

Reply:
There isn't any; Coverity still complains because it can't know that.
However the keyword doesn't hurt and it gets rid of the warning.

We might be able to mark the class as final instead.

Reply:
All well, this is all a bit above my coding grade. 🤷‍♂️ However you have cleared things up for me so it does seem reasonable.